projects/mci/inc/motor_can.h

```diff
@@ -54,46 +51,42 @@ typedef uint32_t MotorCanFrameId;
 #define MOTOR_CAN_RIGHT_BUS_MEASUREMENT_FRAME_ID (0x82u)
 #define MOTOR_CAN_RIGHT_VELOCITY_MEASUREMENT_FRAME_ID (0x83u)
 
-/* Frame lengths in bytes. */
+// Frame lengths in bytes.
 #define MOTOR_CAN_DRIVE_COMMAND_LENGTH (8u)
 
-/* Extended or standard frame types. */
+// Extended or standard frame types.
 #define MOTOR_CAN_DRIVE_COMMAND_IS_EXTENDED (0)
 
-/* Signal choices. */
+// Signal choices.
+
+// Signals in message Right_DriveCommand.
+
+// All signal values are as on the CAN bus.
 
-/**
- * Signals in message Right_DriveCommand.
- *
- * All signal values are as on the CAN bus.
- */
 typedef struct MotorCanDriveCommand {
-  /**
-   * Desired motor velocity set point in metres/second.
-   *
-   * Range: -
-   * Scale: 1
-   * Offset: 0
-   */
+  // Desired motor velocity set point in metres/second.
+
+  // Range: -
+  // Scale: 1
+  // Offset: 0
+
   float motor_velocity;
```

Comment:
IMO there shouldn't be all the blank lines in the comments, please remove the blank lines just before each of the variables/structs, and either put a `//` on the blank lines within comments or remove the blank lines.